src/runtime/getTokenId.ts

```diff
@@ -0,0 +1,13 @@
+import { NetworkId } from '../api/networkId'
+
+export function getTokenId({
+  networkId,
+  isNative,
+  address,
+}: {
+  networkId: NetworkId
+  isNative: boolean
+  address?: string
+}) {
+  return `${networkId}:${isNative ? 'native' : address?.toLowerCase()}`
+}
```

Comment:
Nothing to do now, but I think it's the 2nd time this week I notice we spread this helper to multiple repos.
Maybe we should have a small lib to avoid the duplication.

Reply:
yeah makes me nervous every time lol. It would be easy to see some drift, particularly in whether the address is lowercased or not